src/features/useFetch.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+
+function useFetch(urls) {
+    const [data, setData] = useState(null);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const responses = await Promise.all(urls.map(url => fetch(url)));
```

Comment:
how come you are using browser FETCH here and using AXIOS in some other places???

Reply:
Wanted to know how to use FETCH

Reply:
Should I change it to AXIOS?